junifer/markers/utils.py

```diff
@@ -156,3 +157,34 @@ def _correlate_dataframes(
         .corr(method=method)  # type: ignore
         .loc["df2", "df1"]
     )
+
+
+def _voxelwise_tsnr(input: Dict) -> Dict:
```

Comment:
no need to have this in utils, it can be in the base marker, making the TNSR marker self-contained.

Reply:
Updated.